tests/integration/launch_container.sh

```diff
@@ -112,17 +111,21 @@ set +x
 
 echo "Launching ${container_id}..."
 
-total=24
+total_retries=24
 if $is_llm; then
-  echo "extra sleep for 2 min on LLM models"
-  total=60
+  total_retries=60
   if [[ "$platform" == *"inf2"* ]]; then
-    total=80
+    total_retries=80
+  fi
+  if [[ "$platform" == *"trtllm"* ]]; then
+    total_retries = 100
+    echo "extra sleep of 5 min for trtllm compilation"
```

Comment:
this is not correct, we are not sleeping for 5min, instead we will keep retrying for 1500s, and that means 27min